Form-Controls/index.html

```diff
@@ -1,28 +1,129 @@
+  <title>Document</title>
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
+  <title>Della Bella Form</title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Azeret+Mono:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Azeret+Mono:ital,wght@0,100..900;1,100..900&family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap" rel="stylesheet">
 </head>
 <body>
+      <!--nav menu-->
+  <nav class="menu"> 
+    <div>
+      <img class="logo" src="img/logo.png" alt="logo river west brand">
+    <ul class="menu-itens">
+      <li><a href="#">Home</a></li> 
+      <li><a href="#">Men</a></li> 
+      <li><a href="#">Woman</a></li> 
+      <li><a href="#">Kids</a></li> 
+    </ul>
+  </div>
+  </nav>
+  <hr class="divider"> </hr>
+   <!--end nav menu-->
+
+ <!--HEADER-->
   <header>
-    <h1>Product Pick</h1>
+    <h1>NEW FALL COLLECTION</h1>
+    <h2> LOGIN TO OUR EXCLUSIVE CLUB OFFERS</h2>
+    <form class="log-user">
+      <label for= "name" type="text">Name</label>
+      <input required minlength="2" type="text" id="name" placeholder="Name and Last Name"><br><br>
+       <label for="email">E-mail</label>
+       <input required  type="email" id="email" name="email" placeholder="Please confirm your email"><br>
+
+       <button class="btn" type="submit" name="submit" value="sub">Submit</button>
+    </form>
+  
   </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+   <!--END HEADER-->
+
+<main>
+    <!--  
+          a valid name. 
+          require a valid email.
+          require one colour from a defined set of 3 colours.
+          require one size from a defined set of 6 sizes.
+          require one date from a constrained date range.-->
+
+        <section  class="product-card">
+          <div class="image-product">
+            <img class="product-img" src="img/jumper1.jpg" alt="jumper cashmere yellow">
+          </div>
+          <section>
+          
+             <!-- MUST HAVE THE same NAME TO allowed only 1 been selected-->
+             
+             <form class="product-features">
+              <p class="product-name">PULLOVER AGHATA</p>
+              <p class="product-price">£19.90</p>
+            <legend>Color:</legend>
+
+             <!--COLURS !!!REMEMBER IMPORTANT THE SAME NAME SO ONLY ONE CAN BE CHOSEN!!!-->
```

Comment:
Yes, but also grouping with fieldset https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/

Reply:
done the changes. 
<form class="product-features">
 <fieldset>
  <legend>Color:</legend>